.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+name: "Close stale issues"
+on:
+  schedule:
+    # hourly
+    - cron: "0 * * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+          only-labels: "More Info Requested"
+          stale-issue-message: "Greetings! It looks like this issue hasn't been active in longer than a week. Because it has been a while since the last update on this, and in the absence of more information, we will be closing this issue soon. Please feel free to provide a comment to prevent automatic closure, or if the issue is already closed, please feel free to open a new one."
+          stale-issue-label: "Closing Soon"
+
+          # Issue timing
+          days-before-stale: 7
+          days-before-close: 4
```

Comment:
I'm just wondering if 7 days is enough -- people can just be on vacation for 2 weeks and come back and realize their issue got closed :)  And then on the flip side, IIUC we would be expected to remove the "More Info Requested" label within 7 days after they respond so that the issue doesn't get marked as stale again.  I think in the Spinnaker community, we do 45 days to become stale, and then another 45 days before closure -- that might be TOO long ;)

Reply:
Timing updated to:

```
days-before-stale: 45
days-before-close: 15
```

I erred on the side of being more forgiving, we can always tighten the numbers in future.